common/ferrostar/src/routing_adapters/snapshots/ferrostar__routing_adapters__graphhopper__tests__request_body_with_custom_profile.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: ferrostar/src/routing_adapters/graphhopper.rs
+expression: "generate_body(USER_LOCATION, WAYPOINTS.to_vec(),\nSome(r#\"{\n            \"ch.disable\": true,\n            \"custom_model\": {\n                \"distance_influence\": 15,\n                \"speed\": [\n                    {\n                        \"if\": \"road_class == MOTORWAY\",\n                        \"limit_to\": \"100\"\n                    }\n                ]\n            }\n        }\"#))"
+---
```

Comment:
@karussell can you give these snapshots a quick check? These are generated in the routing adapter tests I just added. Below is JSON that would be generated and sent to the server. I couldn't find any obvious docs on the navigate API but looking through the code, I think this is OK...

Reply:
> What is the purpose of these snapshots?

Basically a sanity check that will check that we're generating the API request correctly. If we change anything about how it's generated by accident, the snapshot test will fail until it's updated.

Sort of like a unit test that writes itself ;)